docs/technical_reference/unit_models/gac.rst

```diff
@@ -237,15 +237,105 @@ if ``surface_diffusion_coefficient_type`` is set to ``calculated``:
+   "Cost to regenerate spent GAC adsorbent by an offsite regeneration facility", ":math:`C_{op,regen}`", "gac_regen_cost", ":math:`$/year`"
+   "Cost to makeup spent GAC adsorbent with fresh adsorbent", ":math:`C_{op,makeup}`", "gac_makeup_cost", ":math:`$/year`"
+
+Capital Cost Calculations
++++++++++++++++++++++++++
+
+Capital costs are determined by the summation of three costing terms. Each term is is calculated by a one parameter
+(different for each term) function considering economy of scale.
+
+    .. math::
+
+        C_{cap,tot} = C_{cap,bed}+C_{cap,carbon}+C_{cap,other}
+
+Contactor and GAC adsorbent capital costs are estimated using functions and parameters reported in US EPA, 2021. Contactors
+are assumed to be carbon steel pressure vessels with plastic internals and are determined as a polynomial function of
+individual contactor volume. The unit cost per kilogram of GAC adsorbent needed is calculated using an exponential
+function. A maximum reference mass is imposed in the costing method to define a best available price where above
+this required charge, the price would no longer be discounted. Other process costs (vessels, pipes, instrumentation,
+and controls) included in the US EPA, 2021 model are aggregated into a separate term. The parameters for the power law
+function with respect to the total system contactor volume were regressed using results from the US EPA, 2021 model.
+
+    .. math::
+
+        & C_{cap,bed} = \left( N_{op}+N_{red} \right)\left( x_0+x_1\left( \frac{V}{N_{op}} \right)+x_2\left( \frac{V}{N_{op}} \right)^2+x_3\left( \frac{V}{N_{op}} \right)^3 \right) \\\\
+        & C_{carbon} = y_0e^{y_1M_{GAC}^{ref}} \\\\
+        & C_{cap,carbon} = C_{carbon}M_{GAC} \\\\
+        & C_{cap,other} = z_0\left( \left( N_{op}+N_{red} \right)\frac{V}{N_{op}} \right)^{z_1}
+
+Note that given the the ability to alter the parameters in these correlations, GAC adsorbent unit costs (:math:`C_{carbon}`)
+may be fixed to a value (:math:`y_0`) by setting :math:`y_1=0`.
+
+Operating Cost Calculations
++++++++++++++++++++++++++++
+
+Operating costs are calculated as the cost to replace spent GAC adsorbent in the contactor beds. Energy for backwash and booster
+pumps are considered negligible compared to the regeneration costs. Since the replacement adsorbent purchases are expected to be
+purchased in bulk at smaller quantities than the initial charge, the cost of fresh GAC adsorbent for makeup has an independent
+cost per unit mass variable, expected to be higher than the initial charge unit cost.
+
+    .. math::
+
+        & C_{op,tot} = C_{op,regen}+C_{op,makeup} \\\\
+        & C_{op,regen} = f_{regen}C_{regen}\dot{m}_{GAC} \\\\
+        & C_{op,makeup} = \left( 1-f_{regen} \right)C_{makeup}\dot{m}_{GAC}
+
+Code Documentation
 -------------------
 
 * :mod:`watertap.unit_models.gac`
+* :func:`watertap.costing.watertap_costing_package.WaterTAPCostingData`
```

Comment:
```suggestion
* :meth:`watertap.costing.watertap_costing_package.WaterTAPCostingData.cost_gac`
```

Reply:
@hunterbarber You should be able to just click on "Commit suggestion" to trigger a new ReadTheDocs build.